packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -159,16 +158,15 @@ class BuildIOSFrameworkCommand extends BuildSubCommand {
       throwToolExit('--output is required.');
     }
 
-    final BuildableIOSApp iosProject = await applicationPackages.getPackageForPlatform(TargetPlatform.ios) as BuildableIOSApp;
-
-    if (iosProject == null) {
-      throwToolExit("Module's iOS folder missing");
+    if (_project.ios.existsSync() == null) {
```

Comment:
shouldn't this be `if (!_project.existsSync()) {`?

Reply:
It's to check that the project even has an iOS directory.  It already checked that the pubspec exists at this point.

Reply:
oh the == null.  Yeah...

Reply:
yeah, my suggestion had a typo...but you know what i mean